docs/modules/record-validation/proc-record-validation.adoc

```diff
@@ -54,7 +54,13 @@ Replace the token `<rule definition>`  in the YAML configuration with either a S
 
 .Example Schema Validation Rule Definition
 
-The Schema Validation rule validates that the key or value matches the identified schema.
+The Schema Validation rule validates that the key or value matches a schema identified by its global ID within an Apicurio Schema Registry.
+
+If the key or value does not adhere to the schema, the record will be rejected.  
+
+Additionally, if the kafka producer has embedded a global ID within the record it will be validated against the global ID defined by the rule.  If they do not match, the record will be rejected.  See the 
+({apicurio-docs}/getting-started/assembly-using-kafka-client-serdes.html#_consumer_schema_configuration)[Apicurio documentation) for details
+of how the global ID could be embedded into the record. The filter supports extracting ID's from either Apicurio `globalId` record header or serialized at the start the content itself (Confluent avro style)
```

Comment:
Do we need the Avro bit?

Reply:
@SamBarker was referring to the Confluent Avro  wireformat https://docs.confluent.io/legacy/platform/4.1.3/schema-registry/docs/serializer-formatter.html#wire-format.  

However, this is not correct.  Apicuiro uses a 8 bytes by default.  (It can use 4 byte for compatibility with Confluent, but this is not the default.

Reply:
I pushed a change.

Reply:
I hadn't twigged the distinction in IDs.